generator/build_test.go

```diff
@@ -103,3 +99,27 @@ func goExecInDir(t testing.TB, target string, args ...string) {
 	p, err := cmd.CombinedOutput()
 	require.NoErrorf(t, err, "unexpected error: %s: %v\n%s", cmd.String(), err, string(p))
 }
+
+var lockLogger sync.Mutex
+
+func discardOutput(w ...io.Writer) func() {
```

Comment:
Can this be merged with the one in utils_test.go?

Reply:
OK. Since these are 2 distinct packages, then I need to factorize inside some internal module. Like so: https://github.com/go-swagger/go-swagger/pull/3037/commits/c1bca457fbeecff841ad8b91fd559be6579f7c38